src/main/java/com/o19s/es/ltr/logging/LoggingSearchExtBuilder.java

```diff
@@ -77,9 +77,11 @@ public static LoggingSearchExtBuilder parse(XContentParser parser) throws IOExce
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-        builder.startObject();
-        builder.field(LOG_SPECS.getPreferredName(), logSpecs);
-        return builder.endObject();
+        //builder.startObject(); //since it's marked as XContentFragment,
+        // so Strings.toString(org.opensearch.core.xcontent.MediaType, org.opensearch.core.xcontent.ToXContent, org.opensearch.core.xcontent.ToXContent.Params, boolean, boolean)
+        // embraces fragment
+        return  builder.field(LOG_SPECS.getPreferredName(), logSpecs);
+        //return builder.endObject();
     }
```

Comment:
not sure if this is always a fragment. I also don't think I trust the test suite to tell us. line coverage is at 45% in com.o19.es.

Reply:
`logSpecs` is a list, so it's hardly might be a fragment 
`LoggingSearchExtBuilder.LogSpec implements ToXContentObject` and has ` builder.startObject();` in `toXContent()` so. they are not fragments.